plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AbstractXmlSourceValidatorTest.java

```diff
@@ -95,32 +99,39 @@ public void testValidate_dynamicWebProject() throws CoreException, ValidationExc
     IncrementalReporter reporter = new IncrementalReporter(null);
     validator.validate(helper, reporter);
     assertEquals(0, reporter.getMessages().size());
+    file.delete(true, null);
   }
 
   @Test
   public void getDocumentEncodingTest() throws CoreException {
-
-    IProject project = appEngineStandardProject.getProject();
+    IProject project = dynamicWebProject.getProject();
     IFile file = project.getFile("testdata.xml");
     file.create(ValidationTestUtils.stringToInputStream(
       APPLICATION_XML), IFile.FORCE, null);
     IDocument document = ValidationTestUtils.getDocument(file);
 
     assertEquals("UTF-8", AbstractXmlSourceValidator.getDocumentEncoding(document));
+    file.delete(true, null);
```

Comment:
May want to put this in a finally block so it happens even if an exception is thrown.